packages/attune/tests/smoke/main.rs

```diff
@@ -0,0 +1,465 @@
+            println!("CLI binary is accessible and executable");
+            println!("✅ Test completed successfully!\n");
+        }
+        Err(e) => {
+            eprintln!("❌ CLI binary test failed: {e}");
+            eprintln!("Make sure the CLI is built and the path is correct.");
+            eprintln!("Current CLI path: {}", config.cli_path);
+            eprintln!(
+                "You can set ATTUNE_CLI_PATH environment variable to specify the correct path."
+            );
+            panic!("CLI binary not accessible");
+        }
+    }
+}
+
+/// Test creating multiple repositories.
+fn test_repo_create_multiple(config: &TestConfig) {
+    let sh = Shell::new().unwrap();
+    set_up_shell_env(&sh, config);
+
+    // Create two different Debian repos for testing.
+    let repo_names = ["debian-test-repo-1", "debian-test-repo-2"];
+    let cli_path = &config.cli_path;
+
+    for repo_name in &repo_names {
+        println!("Creating Debian repo: {repo_name}");
+        
+        let create_result = cmd!(sh, "{cli_path} apt repository create {repo_name}").run();
+
+        match create_result {
+            Ok(_) => println!("✅ Repo '{repo_name}' creation command executed successfully"),
+            Err(e) => {
+                eprintln!("❌ Repo creation failed for '{repo_name}': {e}");
+                panic!("Repo creation failed. Check your environment variables and CLI binary path.");
+            }
+        }
+    }
+    
+    println!("✅ Both repositories created successfully!\n");
+}
+
+/// Test listing repositories to verify both exist.
+fn test_repo_list_multiple(config: &TestConfig) {
+    let sh = Shell::new().unwrap();
+    set_up_shell_env(&sh, config);
+    let expected_repos = ["debian-test-repo-1", "debian-test-repo-2"];
+    let cli_path = &config.cli_path;
+
+    println!("Listing repos to verify both repositories exist...");
+    let list_output = cmd!(sh, "{cli_path} apt repository list").read();
+
+    match list_output {
+        Ok(output) => {
+            println!("Repo list output:");
+            println!("{output}");
+
+            let mut found_repos = Vec::new();
+            for repo_name in &expected_repos {
+                if output.contains(repo_name) {
+                    println!("✅ Repo '{repo_name}' found in list");
+                    found_repos.push(*repo_name);
+                } else {
+                    eprintln!("❌ Repo '{repo_name}' not found in list");
+                }
+            }
+
+            if found_repos.len() == expected_repos.len() {
+                println!("✅ All expected repositories found in list");
+                println!("✅ Test completed successfully!\n");
+            } else {
+                eprintln!("❌ Not all expected repositories found. Expected: {:?}, Found: {:?}", expected_repos, found_repos);
+                eprintln!("Available repos:\n{output}");
+                panic!("Not all created repos found in list");
+            }
```

Comment:
[nit] i recommend choosing either `println!` or `eprintln!` for these tests. contrary to its name, `stderr` isn't really used for _errors specifically_ anymore. 

The typical convention for modern CLI tooling is that `stdout` is for machine readable output and `stderr` is for human readable output.

Reply:
Very interesting, did not know that. Added a TODO for now.